mappings/net/minecraft/scoreboard/IScoreboardCriterion.mapping

```diff
@@ -2,13 +2,14 @@ CLASS none/bdh net/minecraft/scoreboard/IScoreboardCriterion
 	CLASS none/bdh$a Type
 		FIELD a INTEGER Lnone/bdh$a;
 		FIELD b HEARTS Lnone/bdh$a;
-		FIELD c NAME_TO_TYPE_MAP Ljava/util/Map;
+		FIELD c NAME_LOOKUP Ljava/util/Map;
```

Comment:
`TYPE_LOOKUP` if anything, I still suggest `LOOKUP` - would like some input from the other.